lib/app/service/navigation/widget/navigation_service_widget.dart

```diff
@@ -64,20 +68,36 @@ class _NavigationServiceWidgetState extends State<NavigationServiceWidget> {
   void _onRootAppRouteChange(AppRoute oldRoute, AppRoute route) {
     switch ((oldRoute, route)) {
       // When user navigates from onboarding it means that he has created a seed
-      case (AppRoute.onboarding, _):
-
-        /// TODO(alex-a4): HERE is the place to handle onboarding completion
-        /// i.e. show biometric auth dialog
-        break;
+      case (AppRoute.onboarding, AppRoute.wallet):
+        _onboardingToWalletNavigation();
       default:
     }
   }
 
+  Future<void> _onboardingToWalletNavigation() async {
+    // This is a hack, because after navigation from onboarding to wallet
+    // we need wait some time after onboarding screen disappear, on it crashes
+    await Future<void>.delayed(_navigationAwaitDuration);
+
+    // We can't use current context here because it's not contains Navigator
+    final context = NavigationService.navigatorKey.currentState?.context;
+    if (context == null) {
+      return;
+    }
+
+    final biometry = inject<BiometryService>();
+    if (biometry.available) {
+      if (context.mounted) {
+        context.goFurther(AppRoute.enableBiometryAfterOnboarding.path);
```

Comment:
We can move it here, but analyzer will send warning for this line because of using context in async function.
So I'll add this checking to `goFurther` method, but also ill save it here

Reply:
-